docs/source/data-types/data-types.md

```diff
@@ -20,7 +20,8 @@ Database types and their Rust equivalents:
 * `Counter` <----> `value::Counter`
 * `Blob` <----> `Vec<u8>`
 * `Inet` <----> `std::net::IpAddr`
-* `Uuid`, `Timeuuid` <----> `uuid::Uuid`
+* `Uuid` <----> `uuid::Uuid`
+* `Timeuuid` <----> `value::CqlTimeuuid`
```

Comment:
Please add a page for the new `CqlTimeuuid` type, like we already have for `uuid::Uuid` in `uuid.md`